src/main.cpp

```diff
@@ -872,8 +872,7 @@ CAmount GetMinRelayFee(const CTransaction& tx, unsigned int nBytes, bool fAllowF
     CAmount nMinFee = ::minRelayTxFee.GetFee(nBytes);
     nMinFee += GetDogecoinDustFee(tx.vout, ::minRelayTxFee);
 
-    // Dogecoin: Disable free transactions
-    /* if (fAllowFree)
+    if (fAllowFree)
```

Comment:
I've re-enabled free transaction relaying here, which should match behaviour in 1.8